lib/src/config.dart

```diff
@@ -81,11 +83,8 @@ class Config {
   /// Limit of recently used emoji that will be saved
   final int recentsLimit;
 
-  /// The text to be displayed if no recent emojis to display
-  final String noRecentsText;
-
-  /// The text style for [noRecentsText]
-  final TextStyle noRecentsStyle;
+  /// A widget (usually [Text]) to be displayed if no recent emojis to display
+  final Widget noRecentsText;
```

Comment:
What do you think about renaming the attribute to `noRecentsWidget` to make it more consistent with the type? Or is the meaning more obvious by calling it "text" ?

Reply:
Good question. I'm not too sure about this. Maybe we should name it just `noRecents`. I think it's bad code style to include the type in the name. What do you think?

Reply:
Right, sounds good to me ! 👍